<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolStackWidget</class>
 <widget class="QWidget" name="ToolStackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="add_message_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/add_message.svg</normaloff>:/icons/images/add_message.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_message_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/delete_message.svg</normaloff>:/icons/images/delete_message.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_variant_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/add_variant.svg</normaloff>:/icons/images/add_variant.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_variant_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/delete_variant.svg</normaloff>:/icons/images/delete_variant.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mark_start_message_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/first_message.svg</normaloff>:/icons/images/first_message.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mark_error_message_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/error_message.svg</normaloff>:/icons/images/error_message.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="generate_bot_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/generate.svg</normaloff>:/icons/images/generate.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="start_bot_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/start_bot.svg</normaloff>:/icons/images/start_bot.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stop_bot_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/stop_bot.svg</normaloff>:/icons/images/stop_bot.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="generate_logs_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../bot_icons.qrc">
       <normaloff>:/icons/images/logs.svg</normaloff>:/icons/images/logs.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bot_icons.qrc"/>
 </resources>
 <connections/>
</ui>
